angular.json

```diff
@@ -274,7 +274,10 @@
         "lint": {
           "builder": "@angular-devkit/build-angular:tslint",
           "options": {
-            "tsConfig": ["apps/demos/tsconfig.app.json"],
+            "tsConfig": [
+              "apps/demos/tsconfig.app.json",
+              "apps/demos/tsconfig.a11y.json"
```

Comment:
I guess this a11y.json not be part of this changeset? 
It also fails during linting, because it is missing on the CI

Reply:
I was ahead in my changes after rebasing the master I've got the file :) `apps/demos/tsconfig.a11y.json`. So CI should pass now